Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:21.04
+
+RUN apt update && apt install -y ca-certificates
+
+workdir /basilisk
+
+RUN useradd -m -u 1000 -U -s /bin/sh -d /basilisk basilisk && \
+    mkdir -p /basilisk/.local/share/basilisk && \
+    ln -s /basilisk/.local/share/basilisk /data && \
+    chown -R basilisk:basilisk /basilisk
+    
+USER basilisk
+ADD ./basilisk /basilisk/testing-basilisk
```

Comment:
image should be able to run both basilisk and testing basilisk, eg. basilisk should be copied and then testing basilisk hard linked